packages/lesswrong/lib/collections/users/schema.ts

```diff
@@ -2830,7 +2830,20 @@ const schema: SchemaType<DbUser> = {
     canRead: ['guests'],
     hidden: true,
     optional: true
-  }
+  },
+
+  // Giving season fields
+  givingSeason2023DonatedFlair: {
+    type: Boolean,
```

Comment:
Are we gonna add another flair for 'voted'? IMO make this an enum.

Reply:
We are, but I would prefer to keep it as a boolean for simplicity + consistency with the others. Also we may in future want to see independently see who voted and who (we know) donated